transport-native-io_uring/src/main/c/netty_io_uring_native.c

```diff
@@ -333,6 +333,47 @@ static jint netty_io_uring_register_iowq_max_workers(JNIEnv *env, jclass clazz,
      return sys_io_uring_register(ringFd, IORING_REGISTER_IOWQ_MAX_WORKERS, values, 2);
 }
 
+static jlong netty_io_uring_register_buf_ring(JNIEnv* env, jclass clazz,
+                                           int ringFd, unsigned int nentries,
+                                           short bgid, unsigned int flags) {
+    struct io_uring_buf_ring *br;
+	struct io_uring_buf_reg reg;
```

Comment:
Should we better memset these just to be sure everything is cleared correctly and make it more robust ?

Reply:
I think that since br comes from the mmap of MAP_ANONYMOUS, it must be all zero. So I will perform a memset operation on the io_uring_buf_reg reg.